packages/transforms/src/iconToComponent/README.md

```diff
@@ -0,0 +1,4 @@
+# Icon element to `<Icon />` Wildcard component codemod
+
+yarn transform --write --tagToConvert=Link -t ./packages/transforms/src/iconToComponent/iconToComponent.ts '/Users/raphael/Documents/Development/work/Gitstart/client-sourcegraph/client/!(wildcard)/src/\*_/_.{tsx}'
```

Comment:
Before requesting a review, let's update this command by removing user-specific paths.